### ISSUE TITLE
Feature request:  ionic doctor

### ISSUE THREAD

**Issue by gmarziou** [driftyco/ionic-cli#528]
Hello,

In forum, we see so many users who have issues with their environment that it makes me think to node tools like [yo-doctor](https://github.com/yeoman/doctor) and [rff-doctor](https://github.com/rakuten-frontend/rff-doctor) that checks different things about user's environment.

Currently this kind of checking seems to be partly done in different parts: ionic-cli, cordova as part of some workflow. Yo had the good idea to implement it as an external command, so that it could have minimal requirements to run unlike the main yo command.

What about having 'ionic-doctor' ?

**Comment by jbavari** [comment]
Funny you bring this up. A few releases ago, we detected a bug that was resulting from certain users using v0.10.x of Node. Once they upgraded to v0.12.x, it was corrected.

We've put in a simple warning message, however, this idea has been bouncing around the team for quite a while. 

The functionality is already there, we just need a better way to display the data much like `doctor` commands.

I'll see what I can do.
